examples/pytorch/vgae/train.py

```diff
@@ -23,8 +23,8 @@
 parser.add_argument('--hidden2', '-h2', type=int, default=16, help='Number of units in hidden layer 2.')
 parser.add_argument('--datasrc', '-s', type=str, default='dgl',
                     help='Dataset download from dgl Dataset or website.')
-parser.add_argument('--dataset', '-d', type=str, default='pubmed', help='Dataset string.')
-parser.add_argument('--gpu_id', type=int, default=0, help='GPU id to use.')
+parser.add_argument('--dataset', '-d', type=str, default='cora', help='Dataset string.')
+# parser.add_argument('--gpu_id', type=int, default=0, help='GPU id to use.')
```

Comment:
I think you just need to fix that line instead of disabling GPU completely.

Reply:
fix already